templates/index.html

```diff
@@ -39,10 +39,10 @@
                         <a href="/" class="nav-link active"><span data-hover="About">About</span></a>
                     </li>
                     <li class="nav-item">
-                        <a href="/stories" class="nav-link"><span data-hover="Stories">Stories</span></a>
+                        <a href="/stories" class="nav-link"><span data-hover="Student Stories">Student Stories</span></a>
                     </li>
                     <li class="nav-item">
-                        <a href="#align-masters" class="nav-link"><span data-hover="Align Masters">Align Masters</span></a>
+                        <a href="/alignmastersprogram" class="nav-link"><span data-hover="Align Master's Program">Align Master's Program</span></a>
                     </li>
                 </ul>
 
```

Comment:
<img width="1157" alt="Screenshot 2023-09-20 at 3 15 24 PM" src="https://github.com/humans-of-align/humans-of-align-web/assets/62849837/27112545-f064-4551-b011-df220d963bac">